src/components/messages/index.ts

```diff
@@ -3,7 +3,7 @@ import { createContext } from "react";
 export interface IMessage {
   autohide?: number;
   title?: string;
-  text: string;
+  text: React.ReactNode;
```

Comment:
it doesn't look like up to date with master.

Reply:
Oh well, it's because it isn't 🤷‍♂️